srcpkgs/xorgxrdp/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'xrdp'
+pkgname=xorgxrdp
+version=0.2.16
+revision=3
+archs="x86_64"
+build_style=gnu-configure
+hostmakedepends="pkg-config automake autoconf which nasm flex bison libxml2-python3 python3 libtool"
+makedepends="lame-devel opus-devel pixman-devel fuse-devel libjpeg-turbo-devel openssl-devel xorg-server-devel xrdp libdrm-devel"
+depends="lame fdkaac opus pixman fuse libjpeg-turbo xrdp libdrm"
+short_desc="Xorg videodriver for XRDP"
+maintainer="Martijn van Maurik <git@vmaurik.nl>"
+license="Apache-2.0"
```

Comment:
Isn't this one under X11 license?

Reply:
Will check that, I thought it was Apache2 :-)

Reply:
You're right, will fix it.